test_cli_remapping/test/name_maker.py

```diff
@@ -37,8 +37,9 @@ def __init__(self):
 
 if __name__ == '__main__':
     try:
-        with rclpy.init():
-            node = NameMaker()
-            rclpy.spin(node)
+        rclpy.init()
+        node = NameMaker()
+        rclpy.spin(node)
+        rclpy.shutdown()
```

Comment:
This shouldn't be necessary; leaving the `rclpy.init` context should automatically call the shutdown.  If that isn't working, then there is a bug somewhere else.